static/js/themes.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const changeTheme = document.querySelector('#change-theme');
+let currentTheme = null;
+
+// Load themes
+for (let stylesheet of document.styleSheets) {
+	const rules = stylesheet.cssRules;
+	for (let rule of rules) {
+		const match = rule.selectorText?.match(/^\.theme-(\w+)$/)
+		if (match) {
+			const themeOption = document.createElement("option");
+			themeOption.value = rule.selectorText.substring(1);
+			themeOption.innerText = match[1];
+			changeTheme.appendChild(themeOption);
```

Comment:
append, not appendChild

Reply:
appendChild is more appropriate here.
https://dev.to/ibn_abubakre/append-vs-appendchild-a4m

Reply:
Neither is better than the other, and I prefer append over appendChild because of its consistency with other DOM modifying methods.